src/modules/authorization/sagas.ts

```diff
@@ -179,4 +248,30 @@ export function createAuthorizationSaga() {
   }
 }
 
+function getERC20ContractInstance(
```

Comment:
Is there a reason why we're instantiating the contract instead of using the objects from the decentraland-transactions repo?